nurseconnect/constants.py

```diff
@@ -0,0 +1,10 @@
+from django.utils.translation import ugettext_lazy as _
+
+MALE = "m"
+FEMALE = "f"
+UNSPECIFIED = "-"
+GENDERS = {
+    (MALE, _("male")),
```

Comment:
Again, I was only following the method of implementation that was used in molo throughout.